### PR TITLE
added ability to delete todos

### PR DESCRIPTION
**Pull request by tafodinho**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -111,7 +128,13 @@ const renderProjects = () => {
 const renderTodoTitles = (project) => {
   let titleList = '';
   project.todos.forEach((todo) => {
-    titleList += `<li id="${project.id}-${todo.id}">${todo.title}<img class="float-right" id ="${todo.id}" src="../assets/images/icons/bin.svg" height="20px" width="30px" /></li>`;
+    titleList += `<li class"project-todos" 
+                  id="${project.id}-${todo.id}">${todo.title}
```
Comment (SunnyAsar) [inline src/index.js]: desctructuring could be used here to pull out just values

File: src/index.js [hunk 2]
```diff
@@ -171,7 +193,11 @@ const renderTodo = (todo) => {
           </p>
           <div class="clearfix todo-item-footer">
               <h6 class="float-left">Project: </h6>
-              <span cclass="float-left">${todo.name}</span>
+              <span cclass="float-left">${projectName}</span>
+          </div>
+           <div class="clearfix todo-item-footer">
```
Comment (SunnyAsar) [inline src/index.js]: is there are real reason we chose not to create forms with the `form` tag?
- there are advantages to that. we could have used the submit event so either button click or enter will trigger a method call.
- we can call form rest on it to wipe out the fields when we have submitted the form.

Comment (offorma) [reply src/index.js]: I see the advantage you mentioned. We did not think in the direction of a form to be submitted to any backend. Besides we don't need to reset the entire form all the time only a few fileds require a reset. This allows you to continue adding tasks to a project without having to select project all the time except you want a change in project. I think thats a better user experience

**Comment by tafodinho** [comment]
Hi @zenott  i don't quite understand the "User can view all to-dos in each project" part of the task that is to be done because i think i already added ability to view all todos under each project and you can click on a todo to view its details on the right.

**Comment by zenott** [comment]
> Hi @zenott i don't quite understand the "User can view all to-dos in each project" part of the task that is to be done because i think i already added ability to view all todos under each project and you can click on a todo to view its details on the right.

Hi @tafodinho ,

That is fine the way it is. 

Please submit a review request when you think you're ready with the updates.

**Comment by AdnanAfsari** [comment]
Hello @tafodinho @offorma ✋ 

I think you have implemented everything  previous reviewer requested 😺 so Its a 👍 (approved).

Happy Coding

**Pull request merged**